class_defines.py

```diff
@@ -2252,6 +2252,8 @@ class SlvsDistance(GenericConstraint, PropertyGroup):
     )
     flip: BoolProperty(name="Flip", update=update_system_cb)
     draw_offset: FloatProperty(name="Draw Offset", default=0.3)
+    draw_outset: FloatProperty(name="Draw Outset", default=0.0)
+    text_inside: BoolProperty(name="Text Inside", default=True)
```

Comment:
Not sure why this property would be needed, the text is simply drawn based on the draw_outset, right? You could just define a method to check if the text is inside rather than trying to keep this variable in sync.